openreview/conference/templates/areachairWebfield.js

```diff
@@ -258,12 +259,21 @@ var getReviewerGroups = function(noteNumbers) {
 };
 
 var formatData = function(blindedNotes, officialReviews, metaReviews, noteToReviewerIds, invitations, edgeInvitations, tagInvitations, allReviewers, acRankingByPaper, reviewerRankingByPaper) {
-  var uniqueIds = _.uniq(_.reduce(noteToReviewerIds, function(result, idsObj, noteNum) {
+  var uniqueIds = _.uniq(_.concat(_.reduce(noteToReviewerIds, function(result, idsObj, noteNum) {
```

Comment:
I don't understand why you can't just do `var uniqueIds = allReviewers` Doesn't allReviewers contain all the members of the reviewers group?

Reply:
yes, you are right. I thought about that, I didn't want to break anything.

Reply:
I prefer to keep it as it is because it may happen there is a review done by a reviewer that could have been removed from the group. 

The dropdown should only show the members of the reviewer group.